templates/keys.erb

```diff
@@ -0,0 +1,9 @@
+# THIS FILE IS MANAGED BY PUPPET
+# <%= file %>
```

Comment:
I thought this is built-in, because it simply returns the full path and filename. But it is highly probable that I am wrong.